exercises/isogram/src/test/java/IsogramCheckerTest.java

```diff
@@ -41,37 +48,23 @@ public void testIsogramWithHyphen() {
 
     @Ignore("Remove to run test")
     @Test
-    public void testIgnoresMultipleHyphens() {
-        IsogramChecker iso = new IsogramChecker();
-        assertTrue(iso.isIsogram("Hjelmqvist-Gryb-Zock-Pfund-Wax"));
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void testWorksWithGermanLetters() {
-        IsogramChecker iso = new IsogramChecker();
-        assertTrue(iso.isIsogram("Heizölrückstoßabdämpfung"));
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void testIgnoresSpaces() {
+    public void testIsogramWithDuplicatedHyphen() {
         IsogramChecker iso = new IsogramChecker();
-        assertFalse(iso.isIsogram("the quick brown fox"));
+        assertTrue(iso.isIsogram("six-year-old"));
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void testIgnoresSpaces2() {
+    public void testMadeUpNameThatIsAnIsogram() {
         IsogramChecker iso = new IsogramChecker();
         assertTrue(iso.isIsogram("Emily Jung Schwartzkopf"));
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void testDuplicateAccentedLetters() {
+    public void testIsogramDuplicatedCharacterInTheMiddle() {
```

Comment:
Maybe a better name would be something like `testDuplicatedCharacterInTheMiddleIsNotIsogram` or something like that? :)